src/components/Photo/Selfie.js

```diff
@@ -94,13 +99,14 @@ export default class Selfie extends Component<Props, State> {
           /> : null
         }
       >
-        <Timeout seconds={ 10 } onTimeout={ this.handleTimeout } />
+        { !hasCameraError && <Timeout seconds={ 10 } onTimeout={ this.handleTimeout } /> }
         <ToggleFullScreen />
         <FaceOverlay />
         <div className={style.actions}>
           <button
             aria-label={translate('accessibility.shutter')}
             className={style.btn}
+            disabled={hasCameraError}
```

Comment:
Why don't you use the Button component here? Now you have to implement styles for disabled state in two places.

Reply:
Shutter is not a standard button, it's an actual shutter:

![Screenshot 2019-05-07 at 09 59 44](https://user-images.githubusercontent.com/20243687/57287225-f5aeba00-70ae-11e9-9b01-18c72bc64c94.png)

Also, I'm doing as less refactoring as possible as it's out of the scope of the ticket.